src/main/resources/application.properties

```diff
@@ -0,0 +1,16 @@
+spring.application.name=spring-boot-tech-it-easy-controller
+
+#DB_URL=jdbc:postgresql://localhost:5432/conference_app
+#spring.datasource.url=jdbc:postgresql://localhost:5432/conference_app
+spring.datasource.url=jdbc:postgresql://localhost:5432/conference_app
+spring.datasource.username=${DB_username}
+spring.datasource.password=${DB_password}
+
+
+spring.jpa.database-platform = org.hibernate.dialect.PostgreSQLDialect
+spring.jpa.hibernate.ddl-auto = update
+spring.jpa.hibernate.show-sql = true;
+spring.jpa.properties.hibernate.format_sql=true
+
+logging.level.org=WARN
```

Comment:
Dit is handig voor debuggen, "=DEBUG" gebruik je dan, maar je krijgt er wel heel veel output van. "=WARN" is net iets minder output, maar ik weet niet of daar veel nuttige infor tussen zit. Het is leuk om eens te kijken wat de verschillen zijn.